### PR TITLE
FEATURE: Add prototype ``TYPO3.TypoScript:Renderer``

### PR DESCRIPTION
**Pull request by mficzel**
The Renderer object will evaluate to a result using either ``renderer``, ``renderPath`` or ``type`` on the configuration.

### CODE REVIEW

File: TYPO3.TypoScript/Classes/TYPO3/TypoScript/TypoScriptObjects/RendererImplementation.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Matcher object for use inside a "Case" statement
+ */
```
Comment (skurfuerst) [inline TYPO3.TypoScript/Classes/TYPO3/TypoScript/TypoScriptObjects/RendererImplementation.php]: Documentation is wrong here ;)

File: TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/RendererTest.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Testcase for the Case TS object
```
Comment (skurfuerst) [inline TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/RendererTest.php]: Documentation

**Comment by mficzel** [comment]
@skurfuerst ... thank you for the nitpicks. I pushed an update.

**Comment by gerhard-boden** [comment]
@skurfuerst could you please remove your `-1`

**Comment by skurfuerst** [comment]
Thanks!

**Pull request merged**